web/installer

```diff
@@ -52,10 +52,12 @@ function process($argv)
     }
 
     if ($ok || $force) {
-        installComposer($version, $installDir, $filename, $quiet, $disableTls, $cafile, $channel);
-        showWarnings($warnings);
-        showSecurityWarning($disableTls);
-        exit(0);
+        $installer = new Installer($quiet, $disableTls, $cafile);
+        if ($installer->run($version, $installDir, $filename, $channel)) {
```

Comment:
Correct. This PR is the last in a series to make the installer code a little more dev-friendly, but it is only the first stage as I see it. The next is to get it in sync with the main codebase.